extensions/amp-story/1.0/amp-story.js

```diff
@@ -1632,6 +1635,10 @@ export class AmpStory extends AMP.BaseElement {
         actions.execute(this.sidebar_, 'open', /* args */ null,
             /* source */ null, /* caller */ null, /* event */ null,
             ActionTrust.HIGH);
+      } else if (this.sidebar_
+                && !sidebarState
```

Comment:
nit: this seems like it could fit on the previous line

Reply:
Done.